src/libs/error.js

```diff
@@ -9,6 +9,8 @@ export const errorStore = Utils.atom([])
 const addError = item => errorStore.update(state => _.concat(state, [item]))
 
 export const reportError = async (title, obj) => {
+  if (obj instanceof DOMException && obj.name === 'AbortError') return
```

Comment:
Rather than dealing with this here, I wonder if should wrap the underlying promises returned from 'fetch', and prevent them from rejecting when the call is aborted. That way, we'll avoid running any callbacks, including ones that don't call `reportError`.